diff --git a/3rdparty/qtdbusextended-0.0.3/src/src.pro b/3rdparty/qtdbusextended-0.0.3/src/src.pro
index e9c2c28..27b805a 100644
--- 3rdparty/qtdbusextended-0.0.3/src/src.pro
+++ 3rdparty/qtdbusextended-0.0.3/src/src.pro
@@ -31,15 +31,15 @@ INSTALL_HEADERS = \
     dbusextended.h \
     dbusextendedabstractinterface.h
 
-target.path = $$[QT_INSTALL_LIBS]
+target.path = $$PREFIX/lib
 headers.files = $$INSTALL_HEADERS
-headers.path = $$[QT_INSTALL_HEADERS]/DBusExtended
+headers.path = $$PREFIX/include/DBusExtended
 prf.files = $${TARGET}.prf
 prf.path = $$[QMAKE_MKSPECS]/features
-INSTALLS += target headers prf
+INSTALLS += target headers
 
-QMAKE_PKGCONFIG_LIBDIR = $$target.path
-QMAKE_PKGCONFIG_INCDIR = $$headers.path
+QMAKE_PKGCONFIG_LIBDIR = \"$$target.path\"
+QMAKE_PKGCONFIG_INCDIR = \"$$headers.path\"
 QMAKE_PKGCONFIG_DESTDIR = pkgconfig
 QMAKE_PKGCONFIG_NAME = DBusExtended
 QMAKE_PKGCONFIG_REQUIRES += Qt5Core Qt5DBus
